src/workerd/api/node/tests/sidecar-supervisor.mjs

```diff
@@ -60,13 +63,53 @@ function runSidecar(cmd) {
   return { promise, proc };
 }
 
+function getRandomLoopbackHost() {
+  // Pick a random address in 127.0.0.0/8.
+  // Range is chosen not to use network address, gateway address, or broadcast address.
+  return `127.${crypto.randomInt(2, 255)}.${crypto.randomInt(2, 255)}.${crypto.randomInt(2, 255)}`;
```

Comment:
What happens if this loopback host is taken? We don't check it anywhere

Reply:
If a host is taken, we'll get some random ports on it anyway. This will increase the likelihood of collision but will otherwise work.